gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionSha256Sum=7faa7198769f872826c8ef4f1450f839ec27f0b4d5d1e51bade63667cbccd205
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.8.3-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.9-bin.zip
```

Comment:
Why the is `distributionSha256Sum` missing?

Reply:
The IDEA issue with it has been fixed in the meantime. I'll re-add it when upgrading to Gradle 7.

Reply:
I just ran `./gradlew wrapper --gradle-version=6.9` -- this was the result.

Reply:
@szpak was referring to the additional `--gradle-distribution-sha256-sum` option (cf. https://docs.gradle.org/current/userguide/gradle_wrapper.html#configuring_checksum_verification)

Reply:
yes, i know -- I'm just saying that the wrapper command seems to have removed that line. Not sure if it's not required....

Anyway , per that link you posted, it seems like you could add a GitHub Action to validate the wrapper checksum. Might be a good idea.

Reply:
@nhajratw The following command is better:
```
./gradlew wrapper --gradle-version 7.0.2 --distribution-type bin --gradle-distribution-sha256-sum 0e46229820205440b48a5501122002842b82886e76af35f0f3a069243dca4b3c
```
(with checksum available [here](https://gradle.org/release-checksums/))

Reply:
> Anyway , per that link you posted, it seems like you could add a GitHub Action to validate the wrapper checksum. Might be a good idea.

We have one, so maybe it even did that :)
https://github.com/gradle-nexus/publish-plugin/actions/workflows/gradle-wrapper-validation.yml

Reply:
> @szpak was referring to the additional --gradle-distribution-sha256-sum option

I missed that your command Marc. Thanks.

Reply:
This action validates the checksum of the wrapper JAR that is checked in in `gradle/wrapper`. The `distributionSha256Sum` is about validating the checksum of the Gradle distribution that is downloaded by the wrapper.